File: DanceSequence/ClientApp/src/app/model/addSequence.ts
import { Move } from '../model/move';
export class AddSequence {
    public name: string;
    public ids: number[];
    public userId: number;

    constructor(name: string, moves: Move[], userId: number) {
       this.name = name;
       this.ids = [];
       this.userId = userId;
       for(let move of moves)
       {
        this.ids.push(move.id);
       }
    }
}

File: DanceSequence/Models/Add/AddSequence.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DanceSequence.Models.Add
{
    public class AddSequence
    {
        public string Name { get; set; }
        public List<int> Ids { get; set; }
        public int UserId { get; set; }
    }
}
